js/test/analyze.test.js

```diff
@@ -0,0 +1,15 @@
+import analyze from '../analyze';
+
+const tester = analyze([1, 2, 3, 4, 5]);
+
+test('It should return a resulting object', () => {
+  expect(typeof tester).toBe('object');
+});
+
+test('The object should have 4 properties', () => {
+  expect(Object.keys(tester).length).toEqual(4);
+});
+
+test('The properties should be average, min, max, and length', () => {
```

Comment:
- [ ] You are missing the tests checking if the values of 'average', 'min', 'max', 'length' are correct.